engine/access/state_stream/backend/handler.go

```diff
@@ -601,22 +800,45 @@ func (h *Handler) SubscribeAccountStatusesFromLatestBlock(
 	stream executiondata.ExecutionDataAPI_SubscribeAccountStatusesFromLatestBlockServer,
 ) error {
 	// check if the maximum number of streams is reached
-	if h.StreamCount.Load() >= h.MaxStreams {
+	if h.activeStreamCount.Load() >= h.maxStreamCount {
 		return status.Errorf(codes.ResourceExhausted, "maximum number of streams reached")
 	}
 
-	h.StreamCount.Add(1)
-	defer h.StreamCount.Add(-1)
+	h.activeStreamCount.Add(1)
+	defer h.activeStreamCount.Add(-1)
 
 	statusFilter := request.GetFilter()
-	filter, err := state_stream.NewAccountStatusFilter(h.eventFilterConfig, h.chain, statusFilter.GetEventType(), statusFilter.GetAddress())
+	filter, err := state_stream.NewAccountStatusFilter(
+		h.eventFilterConfig,
+		h.chain,
+		statusFilter.GetEventType(),
+		statusFilter.GetAddress(),
+	)
 	if err != nil {
-		return status.Errorf(codes.InvalidArgument, "could not create account status filter: %v", err)
+		return status.Errorf(
+			codes.InvalidArgument,
+			"could not create account status filter: %v",
+			err,
+		)
```

Comment:
I've left a number of comments across several PRs about unnecessary changes breaking code out onto separate lines. Uliana has as well. I'm guessing you just have a setting in your IDE that's doing it. 

Please adjust your settings to avoid making unrelated spacing only changes unless you are already modifying the code and it's fixing a line with a large list of arguments or that is very long (120+ characters). It makes reviews more difficult because I have to compare each of these lines to see what's different. In many cases, I find that it makes the resulting code less readable because it takes up more vertical room. 

For instance, about 25% of lines changed in this fail are required. the rest add review overhead.

```suggestion
		return status.Errorf(codes.InvalidArgument, "could not create account status filter: %v", err)
```